src/data/ACTIONS/layers/patch6.1.ts

```diff
@@ -9,5 +10,11 @@ export const patch610: Layer<ActionRoot> = {
 		GRIT: {cooldown: 3000},
 		IRON_WILL: {cooldown: 3000},
 		ROYAL_GUARD: {cooldown: 3000},
+
+		//SAM 6.1 action changes:
+		HISSATSU_KAITEN: {id: SHARED.UNKNOWN.id}, //Kaiten was removed. But is job critical for pre-6.1 analysis.
+		// Potency buffs, very important, breaks postionals without them.
+		GEKKO: {potency: 330},
+		KASHA: {potency: 330},
```

Comment:
I think more needs to be done to fix the module. The Positional logic relies on the ratio of Combo potency (no positional) to No-combo potency (also no positional) which was also updated.

In both Gekko and Kasha's case, it seems to be:
100 -> 120
320 -> 330.

This fixes only the 330 part, but not the 120 part.

In https://github.com/xivanalysis/xivanalysis/blob/endwalker/src/parser/core/modules/Positionals.tsx I chose to model potencies as a list of potencies with a repeated field of modifiers, which might be necessary if you want to start adding potencies to actions.

Reply:
I added a "noComboPotency" to the Actions to handle that issue. if we want to fix the actual postionals code to handle it instead, let this be a bandaid.

Reply:
Might be better to fix it properly and just wait for that PR, so something like potency being optional still but typed as `number | PotencyConfig` or some such where the latter is an interface describing modifiers like base, combo, proc, etc. Sorta like how combos work except more flexible and backwards compatible.